src/packages/end-to-end/src/types.ts

```diff
@@ -0,0 +1,3 @@
+export type Album = {
+	id: number;
```

Comment:
I see this id is a number, why? SQlLite?

Reply:
yeah it is an integer in the sqlite db